docs/rst/user_guide/atmosphere/intro.rst

```diff
@@ -0,0 +1,8 @@
+.. _sec-atmosphere-intro:
```

Comment:
I'd like to change conventions and use `index.rst` as the landing page for each directory.